examples/models/llama2/CMakeLists.txt

```diff
@@ -52,34 +54,47 @@ find_package(executorch CONFIG REQUIRED)
 # llama_runner library
 add_subdirectory(runner)
 
+set(link_options)
+set(link_libraries)
+
+if(EXECUTORCH_REGISTER_OPTIMIZED_OPS)
+  list(APPEND link_libraries optimized_native_cpu_ops_lib optimized_kernels portable_kernels)
+  list(APPEND link_options
+                      "SHELL:LINKER:--whole-archive \
+                      $<TARGET_FILE:optimized_native_cpu_ops_lib> \
+                      LINKER:--no-whole-archive")
```

Comment:
nit: There is a functions for this in `build/Utils.cmake`, probably better to just use that.

Reply:
but i see that this is just a refactor so ok to do as follow up